config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -345,6 +349,14 @@ use_bazel() (
     fi
 )
 
+additional_deployment_suffices () (
+    local repo="$1"
+
+    case "$repo" in
+        csi-driver-host-path) echo "-prow";;
```

Comment:
would using a suffix like "-singlepod" be more self-descriptive?

Reply:
I preferred to name the additional deployment after their purpose ("testing with Prow - don't use them") instead of what they currently contain ("one pod per sidecar"). My thinking was that the content might change so that it is more than just "single pod", in which case that name would become unsuitable.

But I don't feel strongly about this.

Reply:
My thought was that the purpose of single pod deployment could extend beyond just testing with prow.

Reply:
Oh sorry I thought this suffix was for the new model where all sidecars are in a pod. I guess my preference is still to name it after the deployment model. We have 3 in the repo now: distributed, combinedpod, separatepods?

Reply:
Yes. But the deployment that "normal" users should use doesn't have a suffix ("kubernetes-1.18").

How about "kubernetes-1.18", "kubernetes-1.18-test", "kubernetes-distributed" (no version because there is only one)?

Reply:
I.e. "-test" instead of "-prow"?

Reply:
Sure that's fine. Regarding distributed, should we have the k8s version in it? It's only one now but we're adding 1.21 and 1.22 in the future

Reply:
"distributed" is only used in one job at the moment. I think having just one version is fine, in particular because it works across all supported Kubernetes versions.

Reply:
I've pushed a version with `-test` as suffix. Okay now?

Reply:
I have a feeling that we will need multi k8s versions for distributed deployment for the same reasons why we have multi versions for normal deployments.

For now this is fine.

Reply:
For normal deployments the reason currently is the need to test different snapshotter releases. That's not needed for distributed deployment because it doesn't support snapshotting. But I agree, eventually we might need it. We'll cross that bridge when we get there...